testing/vault/vault.initd

```diff
@@ -2,17 +2,28 @@
 
 VAULT_LOG_FILE="/var/log/${SVCNAME}.log"
 
-command=/usr/sbin/${SVCNAME}
-command_args="${vault_opts}"
-command_background="true"
-start_stop_daemon_args="--user ${SVCNAME}:${SVCNAME}  --stdout $VAULT_LOG_FILE --stderr $VAULT_LOG_FILE"
-pidfile="/run/${SVCNAME}.pid"
+name=vault
+description="Vault is a tool for securely accessing secrets"
+daemon=/usr/sbin/$name
+daemon_user=$name
+daemon_group=$name
 
 start_pre() {
 	checkpath -f -m 0644 -o ${SVCNAME}:${SVCNAME} "$VAULT_LOG_FILE"
 }
 
 depend() {
-        need net
-        after firewall
+	need net
+	after firewall
 }
+
+start() {
+	ebegin "Starting ${SVCNAME}"
+	start-stop-daemon --start --quiet \
+		rm --pidfile /var/run/${name}.pid \
+		--user ${daemon_user} --group ${daemon_group} \
+		-b --stdout $VAULT_LOG_FILE --stderr $VAULT_LOG_FILE \
+		-k 027 --exec ${daemon} -- ${vault_opts}
+	eend $?
+}
```

Comment:
Hm, `-b` is `--background`, so this should do exactly the same thing as the original code. o.O You’ve just moved pidfile to `/var/run` (but that should be symlink to `/run`) and added `-k 027` (that you can simply add to `start_stop_daemon_args=`). Also `rm` is certainly invalid here (`rm --pidfile /var/run/${name}.pid \`), I wonder how it can even work…